src/main/kotlin/no/nav/familie/ks/sak/api/FagsakController.kt

```diff
@@ -84,4 +89,16 @@ class FagsakController(private val fagsakService: FagsakService, private val til
             ?: ResponseEntity.status(HttpStatus.NOT_FOUND)
                 .body(Ressurs.failure(errorMessage = "Fant ikke fagsak på person"))
     }
+
+    @GetMapping(path = ["/{fagsakId}/har-apen-tilbakekreving"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
```suggestion
    @GetMapping(path = ["/{fagsakId}/har-åpen-tilbakekreving"], produces = [MediaType.APPLICATION_JSON_VALUE])
```
Det skal gå fint å bruke æøå i våre endepunkter, denne må sikkert også endres i frontend.

Reply:
fikset det